.github/workflows/wasm-size.yml

```diff
@@ -21,17 +21,15 @@ jobs:
       - name: Checkout PR branch
         uses: actions/checkout@v4
       
-      - uses: dtolnay/rust-toolchain@stable
-
-      - name: 'Install wasm-pack'
-        run: cargo install wasm-pack
+      - uses: cachix/install-nix-action@v24
       
       - name: Build and measure PR branch
         id: measure
         run: |
-          make build-qe-wasm
-          echo "size=$(wc --bytes < query-engine/query-engine-wasm/pkg/query_engine_bg.wasm)" >> $GITHUB_OUTPUT
-          echo "size_gz=$(gzip -cn query-engine/query-engine-wasm/pkg/query_engine_bg.wasm | wc --bytes)" >> $GITHUB_OUTPUT
+          nix build -L .#query-engine-wasm .#query-engine-wasm-gz
+
+          echo "size=$(wc --bytes < ./result/query_engine_bg.wasm)" >> $GITHUB_OUTPUT
```

Comment:
`result` and `result-1` are symlinks that `nix build` creates nad they correspond to wasm and wasm.gz packages respectively.